agent/grpc-internal/client.go

```diff
@@ -52,6 +52,79 @@ const grpcServiceConfig = `
 				"MaxBackoff": "5s",
 				"RetryableStatusCodes": ["RESOURCE_EXHAUSTED"]
 			}
+		},
+		{
+			"name": [
+				{
+					"service": "hashicorp.consul.connectca.ConnectCAService",
+					"method": "WatchRoots"
+				},
+				{
+					"service": "hashicorp.consul.dataplane.DataplaneService",
+					"method": "GetEnvoyBootstrapParams"
+				},
+				{
+					"service": "hashicorp.consul.dataplane.DataplaneService",
+					"method": "GetSupportedDataplaneFeatures"
+				},
+				{
+					"service": "hashicorp.consul.dns.DNSService",
+					"method": "Query"
+				},
+				{
+					"service": "hashicorp.consul.internal.peering.PeeringService",
+					"method": "PeeringList"
+				},
+				{
+					"service": "hashicorp.consul.internal.peering.PeeringService",
+					"method": "PeeringRead"
+				},
+				{
+					"service": "hashicorp.consul.internal.peering.PeeringService",
+					"method": "TrustBundleListByService"
+				},
+				{
+					"service": "hashicorp.consul.internal.peering.PeeringService",
+					"method": "TrustBundleRead"
+				},
+				{
+					"service": "hashicorp.consul.internal.peerstream.PeerStreamService",
+					"method": "StreamResources"
+				},
+				{
+					"service": "hashicorp.consul.serverdiscovery.ServerDiscoveryService",
+					"method": "WatchServers"
+				},
+				{
+					"service": "subscribe.StateChangeSubscription",
+					"method": "Subscribe"
+				},
+				{
+					"service": "partition.PartitionService",
+					"method": "List"
+				},
+				{
+					"service": "partition.PartitionService",
+					"method": "Read"
+				}
+			],
+			"retryPolicy": {
+				"MaxAttempts": 5,
+				"BackoffMultiplier": 2,
+				"InitialBackoff": "1s",
+				"MaxBackoff": "5s",
```

Comment:
If my understanding is right, this would lead to the following schedule: 1s, 2s,4s. 
IS that what we intend? I thought we wanted a 5 time retry.

Reply:
Actually it's a random number between 0 and 5 as per https://github.com/grpc/grpc-proto/blob/master/grpc/service_config/service_config.proto#L145